gulpfile.js

```diff
@@ -158,6 +158,7 @@
                 .src('public/index.html')
                 .pipe(replace(/src="\/lib\/requirejs\/require\.js"/g, 'src="/application.js"'))
                 .pipe(replace(/href="\/css\/main\.css"/g, 'href="/main.css"'))
+                .pipe(replace(/<a href="(https?:\/\/)(.*)" class="cross-site-link">/g, '<a href="$1' + environment + '-$2" class="cross-site-link">'))
```

Comment:
It's meant to skip these steps in production so the URLs in the source file remain as is, but obviously I forgot the skipping logic, will fix. The `package` tasks don't run in development.